apps/stellar/stellar.cpp

```diff
@@ -581,6 +581,9 @@ void _setParser(ArgumentParser & parser)
 
 // TODO(holtgrew): Move this into a SeqAn misc module.
 
+// not supported anymore in vc2015
+// https://msdn.microsoft.com/en-us/library/bb531344.aspx
+#if _MSC_VER < 1900
```

Comment:
I dont know if i understand you correctly. But this _set_output_format function is supported in vs2013 and below, therefore I did not want to change tested code.